Plugins/Cuda/include/Acts/Plugins/Cuda/Seeding/Seedfinder.ipp

```diff
@@ -16,22 +16,43 @@ namespace Acts {
 template <typename external_spacepoint_t>
 Seedfinder<external_spacepoint_t, Acts::Cuda>::Seedfinder(
     Acts::SeedfinderConfig<external_spacepoint_t> config)
-    : m_config(config.toInternalUnits()) {
+    : m_config(config) {
+  // If the size of the regionalParameters doesn't match the number of boundary
+  // fill the remaning region with the default RegionalParameters
+  if (m_config.regionalParameters.size() <
+      (m_config.zboundaries.size() + 1) * (m_config.zboundaries.size() + 1)) {
+    while (m_config.regionalParameters.size() <
+           (m_config.zboundaries.size() + 1) *
+               (m_config.zboundaries.size() + 1)) {
+      m_config.regionalParameters.push_back(
+          Acts::RegionalParameters<external_spacepoint_t>());
+    }
+  }
```

Comment:
This isn't necessary since the cuda Seeding is hard-coded to only use regionalParamters[0], right?

Reply:
Your right I wanted to already add this initialisation mechanism in case someone wanted to implement the use of regionalParamters in the Cuda and Sycl plugin but I could also remove it